.github/workflows/ci.yml

```diff
@@ -117,7 +116,7 @@ jobs:
         shell: bash
         run: |
           DTS_VER="${{steps.dts-ver.outputs.DTS_VER}}"
-          git clone ssh://git@git.3mdeb.com:2222/3mdeb/dts-release-cicd-pipeline.git
+          GIT SSH_COMMAND= 'ssh -F ~/.ssh/config_deploy' git clone ssh://git@git.3mdeb.com:2222/3mdeb/dts-release-cicd-pipeline.git
```

Comment:
Is this correct? There is space between `GIT` and `SSH_COMMAND` and `=` and `'ssh -F...'`